exercises/flatten-array/src/test/java/FlattenerTest.java

```diff
@@ -18,123 +18,57 @@ public void setUp() {
 
     @Test
     public void testFlatListIsPreserved() {
-        assertEquals(
-          asList(0, '1', "two"),
-            flattener.flatten(
-              asList(
-                0,
-                '1',
-                "two")));
+        assertEquals(asList(0, '1', "two"), flattener.flatten(asList(0, '1', "two")));
     }
 
     @Ignore("Remove to run test")
     @Test
     public void testASingleLevelOfNestingWithNoNulls() {
         assertEquals(
-          asList(1, '2', 3, 4, 5, "six", "7", 8),
-          flattener.flatten(
-            asList(
-              1,
-              asList(
-                '2',
-                3,
-                4,
-                5,
-                "six",
-                "7"),
-              8)));
+            asList(1, '2', 3, 4, 5, "six", "7", 8),
+            flattener.flatten(asList(1, asList('2', 3, 4, 5, "six", "7"), 8)));
     }
 
     @Ignore("Remove to run test")
     @Test
     public void testFiveLevelsOfNestingWithNoNulls() {
         assertEquals(
-          asList(0, '2', 2, "three", '8', 100, "four", 50, "-2"),
-          flattener.flatten(
-            asList(
-              0,
-              '2',
-              asList(
-                asList(
-                  2,
-                  "three"),
-                '8',
-                100,
-                "four",
-                singletonList(
-                  singletonList(
-                    singletonList(
-                      50)))),
-              "-2")));
+            asList(0, '2', 2, "three", '8', 100, "four", 50, "-2"),
+            flattener.flatten(asList(0, '2', asList(asList(2, "three"), '8', 100, "four",
```

Comment:
When a line is too long a would prefer for either all the arguments to go on a new line if they fit, or for each argument to go on a new line. So in this case either: 
```
flattener.flatten(
    asList(0, '2', asList(asList(2, "three"), '8', 100, "four", singletonList(singletonList(singletonList(50)))), "-2")
)
```

or
```
flattener.flatten(
    asList(
        0,
        '2',
        asList(asList(2, "three"), '8', 100, "four", singletonList(singletonList(singletonList(50)))),
        "-2"
    )
)
```

Especially in an exercise like this when it's very easy to get confused about which arguments go with which functions. Does that make sense? 🙂

Reply:
done

Reply:
done

Reply:
done

Reply:
done